sessionctx/variable/tidb_vars.go

```diff
@@ -113,8 +113,8 @@ const (
 	// If the query has a LIMIT clause, high concurrency makes the system do much more work than needed.
 	TiDBDistSQLScanConcurrency = "tidb_distsql_scan_concurrency"
 
-	// tidb_opt_insubquery_unfold is used to enable/disable the optimizer rule of in subquery unfold.
-	TiDBOptInSubqUnFolding = "tidb_opt_insubquery_unfold"
+	// tidb_opt_insubquery_rewriting is used to enable/disable the optimizer rule of rewriting IN subquery.
+	TiDBOptInSubqRewriting = "tidb_opt_insubquery_rewriting"
```

Comment:
how about:
```
s/tidb_opt_insubquery_rewriting/tidb_opt_insubquery_to_innerjoin/
s/TiDBOptInSubqRewriting/TiDBOptInSubqueryToInnerJoin/
```

Reply:
The actual form is `inner join together with an aggregation`. So `ToInnerJoin` is not complete but `InnerJoinAndAgg` it too long.